src/util/simplify_expr_int.cpp

```diff
@@ -43,8 +43,8 @@ bool simplify_exprt::simplify_bswap(bswap_exprt &expr)
     mp_integer new_value=0;
 
     //May I suggest to move this out of the loop and instead check that at least as many elements are contained in bytes?
-      // this invariant protects against someone modifying the loop
-        // expression - a bit pointless for it to be outside the loop.
+    // this invariant protects against someone modifying the loop
+    // expression - a bit pointless for it to be outside the loop.
```

Comment:
I'd get rid of this.

Reply:
Apologies @tautschnig - this was an internal note for a colleague for a second opinion. 
I was used to a workflow where I'd push to get CI going, review, then re-add reviewers to flag ready for re-review. 
In future will push after local checks.

Reply:
Oh don't worry, I was just naively assuming this was ready for re-review.